src/test/kotlin/chapter6/exercises/Exercise_6_5.kt

```diff
@@ -6,7 +6,7 @@ import io.kotlintest.shouldBe
 import io.kotlintest.specs.WordSpec
 
 //tag::init[]
-val doubleR: Rand<Double> = TODO()
+val doubleR: Rand<Double> by lazy { TODO() }
```

Comment:
Unfortunately we can't do this, as this code gets included directly into the book by Asciidoctor.
An alternative would be to wrap it in a top level construct like an object which never gets initialized. wdyt?

Reply:
Hey, @marc0der, thanks for quick reply!
Would wrapping `doubleR` in a class work? I have updated the code. 
Could you [take another look](https://github.com/fpinkotlin/fpinkotlin/pull/6/files#diff-fc1dfcabb388b408a05fab9d666df111R8)?